crates/types/src/vid.rs

```diff
@@ -116,22 +117,35 @@ pub struct SmallRangeProofType(
     SmallRangeProof<<UnivariateKzgPCS<E> as PolynomialCommitmentScheme>::Proof>,
 );
 
+#[cfg(feature = "test-srs")]
 lazy_static! {
-    /// SRS comment
-    ///
-    /// TODO use a proper SRS
-    /// https://github.com/EspressoSystems/HotShot/issues/1686
+    /// SRS for testing only
     static ref KZG_SRS: UnivariateUniversalParams<E> = {
         let mut rng = jf_utils::test_rng();
         UnivariateKzgPCS::<E>::gen_srs_for_testing(
             &mut rng,
-            // TODO what's the maximum possible SRS size?
-            checked_fft_size(200).unwrap(),
+            SRS_DEGREE,
         )
         .unwrap()
     };
 }
 
+// By default, use SRS from Aztec's ceremony
+#[cfg(not(feature = "test-srs"))]
```

Comment:
Hmm. I did not properly understand until now your dilemma about "same fn" vs "different fn". 😕 I'm a little uneasy with this violation of the additivity of cargo features. My previous understanding is that `test-srs` is purely additive: it would enable a special test srs used to speed up tests, but it would not remove the aztec srs.

What's the reasoning here? Let me guess: we want to save users who run in test mode the trouble of downloading the aztec srs. Is that all? If so then I don't think that's a good enough reason to violate additivity. Test users can suck up the pain and download aztec. 😛

Reply:
the reason is HotShot seem to want to keep SRS under the same name `static ref KZG_SRS`,
maybe we can have a separate `KZG_SRS_TEST`, which means a separate `fn vid_scheme_test()` etc. and this "extra `_test()` that accepts test srs" pattern will propagate all the way out.

which poison you want to swallow? (I'm fine with both tbh)

Reply:
I've consulted others. The mob has spoken. `test-srs` shall be additive, we'll have a separate `KZG_SRS_TEST`.

Reply:
how does this look [39396a1](https://github.com/EspressoSystems/HotShot/pull/2854/commits/39396a1a1778a636c169cd35143bf3c0a36bacb3)?

note that `vid_scheme_for_test()` is not used anywhere currently, but could be in the future if needed.
atm, all test codes are invoking with production SRS, such as `block_contents.rs::precompute_vid_commitment()` which is invoked by `testing/src/block_builder.rs::build_block()`.

^^ it is obviously also fine to run unit test directly on proper SRS, at least for now it's not a memory or speed concern.